src/styles/box/stars.scss

```diff
@@ -0,0 +1,31 @@
+.product-card__stars {
+  width: 100px;
+  display: flex;
+  height: $star-height;
```

Comment:
The variable `$star-height` is used here, but it is not defined in this file. Ensure that `$star-height` is defined in a file that is imported before this one, or define it here if it's missing.